service/resources/data/street_suffix.json

```diff
@@ -0,0 +1,21 @@
+{
+    "Avenue": "AV",
```

Comment:
this list probably need to be updated to reflect the mapping from [EAS](https://data.sfgov.org/Geographic-Locations-and-Boundaries/Addresses-with-Units-Enterprise-Addressing-System/ramy-di5m/data). e.g. EAS uses AVE, instead of Avenue